### ISSUE TITLE
Front-end highlighter redesign: Missing screen reader announcements when panel is moved or docked

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#1620]
When the buttons are used to move the panel left or right or to dock or undock a panel, focus is shifted back to the menu trigger button and there is no screen reader announcement that the action was successful.

## Example recording:
https://drive.google.com/file/d/1jN8ubs9UIOyErqYgRETpl9s9wvDxYRN7/view?usp=sharing

## How to fix
Add a hidden screen-reader only div with `role="status" aria-live="polite" aria-atomic="true"` and insert messages into it when the panel is moved, docked, or undocked.
